pkg/common/podgroup/preemptible.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2025 NVIDIA CORPORATION
+// SPDX-License-Identifier: Apache-2.0
+
+package podgroup
+
+import (
+	"fmt"
+
+	"github.com/NVIDIA/KAI-scheduler/pkg/apis/scheduling/v2alpha2"
+	"github.com/NVIDIA/KAI-scheduler/pkg/scheduler/constants"
```

Comment:
Maybe not as part of this PR but the contents of pkg/scheudler/constants/priorities.go should move into pkg/common/constants